### ISSUE TITLE
Where is manual add cam?

### ISSUE THREAD

**Issue by bonus2k** [Jey-Cee/iobroker.onvif#8]
I have two cameras. The driver found only one camera(

![Снимок экрана от 2020-01-04 15-28-30](https://user-images.githubusercontent.com/54687949/71765642-67722880-2f08-11ea-9d29-b81dd95e0645.png)

**Comment by BceBoJlog** [comment]
Yes, how to add a camera by hands? Driver not found my cameras, because they are in other vlan...

**Comment by BceBoJlog** [comment]
@bonus2k can you export your driver's config to file, clear the password and give mi link to config? I'll try edit and import your config, because onvif's driver not finded any cam on my network...

**Comment by Jey-Cee** [comment]
Will come with next version.

**Comment by BceBoJlog** [comment]
Thanks for your time and work, @Jey-Cee
